docs/src/main/sphinx/connector/hive.rst

```diff
@@ -400,6 +400,36 @@ Property Name                                      Description
                                                    not make smart decisions about the query plan.
 ================================================== ============================================================ ============
 
+Storage table properties
+------------------------
+
+.. list-table:: Hive connector storage table properties
```

Comment:
Also needs to have example CTAS queries and maybe link to ALTER TABLE SET PROPERTIES as well. 

Some examples already exist in the doc . move them into this context if appropriate